backend/capellacollab/projects/toolmodels/diagrams/routes.py

```diff
@@ -90,7 +90,7 @@ async def get_diagram(
     except requests.exceptions.HTTPError:
         logger.info("Failed fetching diagram", exc_info=True)
         raise fastapi.HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            status_code=status.HTTP_404_NOT_FOUND,
```

Comment:
Same as above. I think that `HTTP_422_UNPROCESSABLE_ENTITY` is the better option.